roles/jd-server/src/main.rs

```diff
@@ -115,43 +121,63 @@ async fn main() {
         std::time::Instant::now().sub(std::time::Duration::from_secs(60));
 
     if url.contains("http") {
-        let sender_clone = sender.clone();
+        let sender_update_mempool = sender.clone();
         task::spawn(async move {
             loop {
-                let updated_mempool =
+                let result: Result<(), mempool::error::JdsMempoolError> =
                     mempool::JDsMempool::update_mempool(mempool_cloned_.clone()).await;
-                if let Err(err) = updated_mempool {
+                if let Err(err) = result {
```

Comment:
I would leave `updated_mempool` here (result is too much generic imo)

Reply:
Do you like `update_mempool_result`? :-)